test/items_repo_test.rb

```diff
@@ -31,7 +31,7 @@ def test_it_finds_by_unit_price
     item2 = Item.new(:unit_price => 45098)
     items = [item, item2]
     repo = ItemsRepo.new(items)
-    assert_equal item, repo.find_by_unit_price(67076)
+    assert_equal item, repo.find_by_unit_price(670.76)
```

Comment:
The problem with just doing this is you are kind of "cheating" the test. We are expected to use the `BigDecimal` class like it shows in the spec:
```Ruby
    describe ".find_by_unit_price" do
      it "can find one record" do
        item = engine.item_repository.find_by_unit_price BigDecimal.new("935.19")
        expect(item.name).to eq "Item Alias Nihil"
      end
    end
```
So we need to implement our tests that way in order to prove it is working. We need to call 
```Ruby
repo.find_by_unit_price(BigDecimal.new("935.19"))
```
to match the spec. This is the big problem. It is difficult to know how to do this, and I've given up on it in a way. If your code here passes both our tests and the spec, then fuck it, might as well merge. But get rid of the changes that were already in the `spec-harness` branch.